### PR TITLE
feat: Adding API v2 Bigquery export samples

### PR DESCRIPTION
**Pull request by rafaelMurata**
## Description

b/335222583

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by Sita04** [comment]
@rafaelMurata @RajviM I see this PR refactors the existing v1 samples. Acc to previous discussions, IIUC, we are trying to create new samples in addition to the existing ones right?

**Comment by rafaelMurata** [comment]
@Sita04  @RajviM
I did not participate in the IIUC discussions. I removed the previous samples due to the fact that v1 is no longer available. But if you prefer, I can return to v1 classes, no problem.

com.google.api.gax.rpc.InvalidArgumentException: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: This API is no longer available. Please use API V2.

**Comment by Sita04** [comment]
@rafaelMurata Kindly address the test failures.

**Comment by rafaelMurata** [comment]
> @rafaelMurata Kindly address the test failures.

Done. 
Please try again.

File: security-command-center/snippets/src/main/java/vtwo/bigquery/CreateBigQueryExport.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  // Create export configuration to export findings from a project to a BigQuery dataset.
+  // Optionally specify filter to export certain findings only.
+  public static void createBigQueryExport(String organizationId, String location,
+      String projectId, String filter, String bigQueryDatasetId, String bigQueryExportId)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/bigquery/CreateBigQueryExport.java:62]:
```suggestion
    // once, and can be reused for multiple requests.
```

Comment (Sita04) [reply security-command-center/snippets/src/main/java/vtwo/bigquery/CreateBigQueryExport.java:62]: Applies to multiple places.

File: security-command-center/snippets/src/main/java/vtwo/bigquery/UpdateBigQueryExport.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+  // Updates an existing BigQuery export.
+  public static void updateBigQueryExport(String organizationId, String location, String filter,
+      String bigQueryExportId)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      // optionally BigQueryExportName or String can be used
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/bigquery/UpdateBigQueryExport.java:60]:
```suggestion
      // Optionally BigQueryExportName or String can be used
```

**Review by Sita04 — APPROVED** [review]
Great work on the snippets. Kindly merge once all tests pass.

**Comment by rafaelMurata** [comment]
@Sita04 
Please run kokoro again. Thanks.

**Comment by rafaelMurata** [comment]
@Sita04 
I had the same problem when trying to create a bigquery dataset in my machine. so I changed it to be the same way as in v1, but apparently I have the same problem. I suspect it is permission in kokoro. Is there anything I can do?

[ERROR] vtwo.BigQueryExportIT  Time elapsed: 1.817 s  <<< ERROR!
com.google.api.gax.rpc.PermissionDeniedException: io.grpc.StatusRuntimeException: PERMISSION_DENIED: The caller does not have permission
Caused by: io.grpc.StatusRuntimeException: PERMISSION_DENIED: The caller does not have permission

**Pull request closed without merging**